examples/cloud-auth/project1/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You need to change the directory to `auth-cloud` to keep the pattern and remove `project1`